OKTA-544341 Update locale default profile attribute users api (#3713)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/users/index.md

| countryCode         | country name component of user's address (`country`)                                                                                 | String     | TRUE            | FALSE    | FALSE      |             |             | [ISO 3166-1 alpha 2 "short" code format](https://tools.ietf.org/html/draft-ietf-scim-core-schema-22#ref-ISO3166) |
| postalAddress       | mailing address component of user's address                                                                                          | String     | TRUE            | FALSE    | FALSE      |             |             |                                                                                                                  |
| preferredLanguage   | user's preferred written or spoken languages                                                                                         | String     | TRUE            | FALSE    | FALSE      |             |             | [RFC 7231 Section 5.3.5](https://tools.ietf.org/html/rfc7231#section-5.3.5)                                      |

> | locale              | user's default location for purposes of localizing items such as currency, date time format, numerical representations, and so on.         | String     | FALSE (default)            | FALSE    | FALSE      |             |             | See Note for more details.                                                                                       |

Reviewer: ```suggestion
| locale              | user's default location for purposes of localizing items such as currency, date time format, numerical representations, and so on         | String     | FALSE            | FALSE    | FALSE      |             |             | See [Locale format](#locale-format).                                                                                       |
```
I'm converting the Note to an anchored section for linking. Also, normally we would capitalize the description, even if it's a fragment, but since all the description are lowercase here, I'll leave it. Or we can change it to 
"User's default location for purposes of localizing items such as currency, date time format, numerical representations, and so on."

Author: Thanks, made this fix locally.

Author: Updated the link and new section. Kept the Nullable = FALSE bit because other parameters were also using this format (should go away when we move to Redocly)

> | locale              | user's default location for purposes of localizing items such as currency, date time format, numerical representations, and so on.         | String     | FALSE (default)            | FALSE    | FALSE      |             |             | See [Locale format](#locale-format)details.                                                                                       |

Author: Thanks, got the space and updated all descriptions with a capitalized start.

| managerId           | `id` of a user's manager                                                                                                             | String     | TRUE            | FALSE    | FALSE      |             |             |                                                                                                                  |
| manager             | displayName of the user's manager                                                                                                    | String     | TRUE            | FALSE    | FALSE      |             |             |                                                                                                                  |

> > **Note:** A locale value is a concatenation of the ISO 639-1 two letter language code, an underscore, and the ISO 3166-1 2 letter country code. For example, `en_US` specifies the language English and country US. If not set, this value is `en_US` by default.

Reviewer: ```suggestion
##### Locale format

The `locale` property value is a concatenation of the ISO 639-1 two letter language code, an underscore, and the ISO 3166-1 2 letter country code. For example, `en_US` specifies the language English and country US. This value is `en_US` by default.
```

Author: Added this section locally.

Author: Added this locally, thanks.
